src/components/MintDialog/MintDialog.tsx

```diff
@@ -187,7 +209,7 @@ export const MintDialog: FC<{ size?: ButtonProps['size'] }> = ({ size }) => {
           <Success
             resetModal={resetModal}
             txHash={txDetails?.hash ?? ''}
-            closeModal={() => setOpen(false)}
+            closeModal={closeModal}
```

Comment:
what's this about?

Reply:
I made a closeModal function that ensures the `desiredNetwork` gets set back to l2 when the dialog gets closed